portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,62 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Sami's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+
+        <div id="landing-page">
+            <img id="tri" src="tri.png"/>
+            <h1 id="title">Sami Hossain</h1>
+            <div id="social-links">
+                <img src="github.svg" />
+                <img src="linkedin.svg" />
+            </div>
+            <p id="biography">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam molestie metus turpis. Pellentesque efficitur porta nunc, 
+            non posuere tellus condimentum dignissim. Vestibulum facilisis urna ac est sodales, ut molestie diam tristique. Donec quis 
+            mauris a eros consectetur fermentum a vitae justo. </p>
+        </div>
+      
+        <div id="project-page">
+          <h1 id="p-title">Projects</h1>
+          
+          <ul id = "projects">
+            <li> Halite.io </li>
```

Comment:
I noticed your spacing here is a little different from the rest of the file. I'd recommend using a linter to keep everything consistent!

If you are using VS code, this one seems good: https://marketplace.visualstudio.com/items?itemName=hex-ci.stylelint-plus

There's also a library of Github linters here that are relatively easy to set up: https://github.com/collections/clean-code-linters